src/Vizkit3dWorld.hpp

```diff
@@ -133,6 +134,12 @@ class Vizkit3dWorld : public QObject {
     void loadGazeboModelPaths(std::vector<std::string> modelPaths = std::vector<std::string>());
 
     /**
+     * Load the list of the models that we do not want to have the visual nor collision elements
+     * @params list of model's name
+     */
+    void loadExlcudedSdfModels(std::vector<std::string> excludeFromSdf = std::vector<std::string>());
```

Comment:
Typo in method name ("Excluded"). Moreover, there's really no point in having a default argument here (why would one want to call `loadedExcludedModels()` ?)